src/plugins/intel_gpu/src/graph/impls/ocl/convolution.cpp

```diff
@@ -175,6 +175,9 @@ struct convolution_impl : typed_primitive_impl_ocl<convolution> {
                 auto can_swap = [](const kernel_selector::Tensor::DataTensor& dt) -> bool {
                     auto x_channel_idx = static_cast<uint32_t>(kernel_selector::Tensor::DataTensor::Channelndex(dt.GetLayout(),
                                                     kernel_selector::Tensor::DataChannelName::X));
+                    if (x_channel_idx < 0) {
```

Comment:
Isn't this `false positive`? because `x_channel_idx` is unsigned int, So negative case don't occurs

Reply:
I think so since it's static-casting `x_channel_idx` to `unsigned int`, and it needs to suppress in Coverity. Thanks.